tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateProductionData.java

```diff
@@ -575,7 +575,7 @@ private static boolean directoryIsSpecial(String directory) {
     }
 
     private static boolean localeIsSpecial(String localeId) {
-        return localeId.equals("ar") || localeId.startsWith("ar_");
+        return localeId.equals("ar") || (localeId.startsWith("ar_") && !"ar_001".equals(localeId));
```

Comment:
This works for now. In the long run it might be better to check whether a locale is the default content locale (there is info in supplementalMetadata for that), since we could potentially change which locale is the default content locale for `ar`.